### PR TITLE
Fix some typos in French translation

### PR DESCRIPTION
**Pull request by landrok**

### CODE REVIEW

File: po/fr.po [hunk 1]
```diff
@@ -2202,7 +2202,7 @@ msgid ""
 "Space separated list of file patterns used for the find in files dialog (e."
 "g. *.c *.h)"
 msgstr ""
-"Liste de motifs de fichier, séparés par une espace, à utiliser dans le "
+"Liste de motifs de fichier, séparés par un espace, à utiliser dans le "
```
Comment (b4n) [inline po/fr.po]: `une` is technically correct, but OK, `un` is more common nowadays even to refer to the character itself

File: po/fr.po [hunk 2]
```diff
@@ -4412,7 +4412,7 @@ msgid ""
 msgstr ""
 "Tester la correspondance des expressions régulières sur tout le tampon d'un "
 "coup plutôt que ligne à ligne, ce qui permet aux correspondances de "
-"recouvrir plusieurs lignes.  Dans ce mode les caractères de fin de ligne "
+"recouvrir plusieurs lignes.  Dans ce mode, les caractères de fin de ligne "
```
Comment (b4n) [inline po/fr.po]: meh, I don't like unnecessary commas.  but well, if you like.

**Comment by b4n** [comment]
Apart a small remark, LGTM

File: po/fr.po (hunk 2 above)
Comment (landrok) [inline po/fr.po]: _"Le complément circonstanciel est obligatoirement suivi d’une virgule."_ could have been my argument, but, to be honest, it was just in reference to the English sentence L4409 :)

File: po/fr.po (hunk 1 above)
Comment (landrok) [inline po/fr.po]: Quite ambiguous, `un` refers to the space we see whereas `une` refers to the physical chain which will be inserted. I learned one more thing today.

File: po/fr.po (hunk 2 above)
Comment (b4n) [inline po/fr.po]: Fair enough in either case anyway :)
And yeah, in can arguably be useful here.

File: po/fr.po (hunk 1 above)
Comment (b4n) [inline po/fr.po]: yep.  But nowadays `un` is taking over, likely because we're not typesetting manually anymore I guess :)

**Pull request merged**